src/utilities/resolveRelations.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Central place for all the Storyblok relations that we want to resolve
+ * It is used in gatsby-config.ts and page components in src/pages/
+ */
+
+export const resolveRelations = [
+  'sbStoryCard.storyPicker',
+];
```

Comment:
Putting all Storyblok resolve relations in once place. They'll be referenced in several places.